packages/react-dom/src/server/ReactDOMServerFormatConfig.js

```diff
@@ -1877,5 +1880,9 @@ export function writeClientRenderBoundaryInstruction(
   }
 
   writeChunk(destination, boundaryID);
+  if (error) {
+    writeChunk(destination, clientRenderErrorScript1);
+    writeChunk(destination, stringToChunk(error));
```

Comment:
If the error message sneaks in a user space string, that can be used to insert tags like `</script><script>alert('p0wned')`. The approach chosen in https://github.com/facebook/react/issues/23063 won't work here by itself unless we also encode the whole script tag rather than just this string.

Reply:
Maybe if we assume that the rest of the script doesn't contain `</script>` or comments, it's ok but it needs vetting.

Reply:
Hmm, fully escaping it should be fine since this should be a string anyways